lib/install.js

```diff
@@ -28,16 +26,22 @@ var hooks = [
   'post-rewrite',
 ];
 
-function installHooks () {
-  if (isGitProject()) {
-    ensureHooksDirExists();
-    hooks.forEach(install);
+function findGitRoot (directory) {
+  try {
+    return findup.sync(directory || cwd, '.git');
+  } catch(e) {
+    return null;
   }
-  else { warnAboutGit(); }
 }
 
-function isGitProject () {
-  return fs.existsSync(resolve(projectRoot, '.git'));
+function installHooks (directory) {
+  var gitRoot = findGitRoot(directory);
```

Comment:
I believe that this is the reason why you had to bump the `maxcomplexity` up to 3.
We never pass in a `directory` to `installHooks`, so `findGitRoot` also take no parameters and use `cwd` directly. This would bring that function's cyclomatic complexity back to 2, I believe.